index.html

```diff
@@ -105,15 +105,32 @@
     </section>
     <section id="sotd">
       <p>
-        This document is a <strong>work in progress</strong> and is subject to
-        change. It builds on the group's experience on presenting web content on
-        external presentation-type displays, and re-uses patterns and design
+        This document builds on the group's experience on presenting web content
+        on external presentation-type displays, and re-uses patterns and design
         considerations from the Presentation API specification whenever
-        appropriate [[PRESENTATION-API]]. In case of issues or concerns, it is
+        appropriate [[PRESENTATION-API]].
+      </p>
+      <p>
+        Although this document is still a <strong>work in progress</strong> and
+        is subject to change, the Working Group believes that the API surface
+        is stable. Most of the remaining issues listed on the
+        <a href="https://github.com/w3c/remote-playback/issues/">issue
+        tracker</a> are considered minor at this stage, except
+        <a href="https://github.com/w3c/remote-playback/issues/41">issue #41</a>
+        which discusses the set of media playback features that remote playback
+        devices must support and the impact of remote playback on the observable
+        behavior from the controlling page's perspective. The Working Group
+        welcomes feedback on that issue. For other issues or concerns, it is
```

Comment:
Should we note we don't expect API surface changes as a result of resolving this issue though?